initial commit for EA docs (#1702)
denoland/docs: deploy/early-access/usage.md

---
title: "Deno Deploy Early Access Usage Guidelines"
description: "Important limitations, service level expectations, and terms of use for the Deno Deploy Early Access program."
---

> You should be aware of the following limitations when deciding to use Deno
> Deploy EA:

Reviewer: Let's tighten up the language here a bit

```suggestion
As an early access product, Deno Deploy Early Access currently has a number of limitations you should be aware of before using it:
```

- Pro account features do not extend to Deno Deploy EA yet.
- CLI deployment, and deployment from GitHub Actions are not available at this
  time.
- KV, Queues, and Cron are not available in Deno Deploy EA at this time.

:::info

> Deno Deploy EA is an early access product, and as such is not covered by our
> regular service level agreements yet.

Reviewer: Language nit

```suggestion
Deno Deploy EA is an early access product, and as such is not currently covered by
our regular service level agreements.
```

:::

> At this time, the Deno company is already using Deno Deploy EA to host our own
> websites. As may imagine, we are putting significant care into avoiding any
> service interruptions, but as this is a new system, service interruptions can
> not always be avoided.**

Reviewer: Language nits and typos

```suggestion
The Deno company is now using Deno Deploy EA to host our own
websites, and are putting significant efforts into avoiding any
service interruptions, but as this is a new system, minor service interruptions can not always be avoided.
```
